VAMobile/src/components/MilitaryBranchLogo.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react'
+import { Image, ImageStyle, StyleProp } from 'react-native'
+
+import { BranchOfService, BranchesOfServiceConstants } from 'api/types'
+import { useTheme } from 'utils/hooks'
+
+export type MilitaryBranchLogoProps = {
+  branch: BranchOfService
+  width: number
+  height: number
+}
+
+export const MilitaryBranchLogo: FC<MilitaryBranchLogoProps> = ({ branch, width, height }) => {
```

Comment:
Only wandering into this branch out of curiosity of how it sorted out importing directly from the package... but a nitpick: highlighting that the vernacular used by the various branches (granted not uniformly) tends to be they are the "emblem" for the service--in contrast to the "seal" which is reserved for use by the department itself. Some of their brand guides are rather emphatic that their "emblem" is more distinguished than a "logo" as a representation.

Reply:
Good callout Tim. Ironically, I almost named this `MilitaryBranchSeal` but decided not to for that reason. Emblem definitely makes sense, I'll probably rename this to `MilitaryBranchEmblem`